### PR TITLE
[css-fonts] Add tests for standard font families

### PR DESCRIPTION
**Pull request by litherum**
https://github.com/w3c/csswg-drafts/commit/e083fe61cd912abc5c389ea7201b0ae1effe241a

### CODE REVIEW

File: css/css-fonts/standard-font-family-3.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="author" title="Myles C. Maxfield" href="mailto:mmaxfield@apple.com"/>
+<link rel="help" href="https://drafts.csswg.org/css-fonts-4/#standard-font-families"/>
+<link rel="match" href="standard-font-family-3-ref.html"/>
```
Comment (foolip) [inline css/css-fonts/standard-font-family-3.html]: There are a bunch of lint failures in https://tools.taskcluster.net/groups/evSxUmbLRmOr0pxud8icoA/tasks/H_HFJEEURSq4qO-H1iLnFA/runs/0/logs/public%2Flogs%2Flive.log about broken ref links, and this is one place. You can run `./wpt lint` locally to iterate more quickly.

File: css/css-fonts/standard-font-family-10.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="author" title="Myles C. Maxfield" href="mailto:mmaxfield@apple.com"/>
+<link rel="help" href="https://drafts.csswg.org/css-fonts-4/#standard-font-families"/>
+<link rel="match" href="standard-font-family-10-ref.html"/>
```
Comment (svgeesus) [inline css/css-fonts/standard-font-family-10.html]: should be rel="mismatch" and  href="standard-font-family-10-notref.html"

**Review by svgeesus — CHANGES_REQUESTED** [review]
Apart from the specific issues with notref tests, these look good to me.

File: css/css-fonts/standard-font-family-3.html (hunk 1 above)
Comment (litherum) [reply css/css-fonts/standard-font-family-3.html]: Fixed in 2729a10.

File: css/css-fonts/standard-font-family-10.html (hunk 2 above)
Comment (litherum) [reply css/css-fonts/standard-font-family-10.html]: Fixed in 2729a10.

**Review by svgeesus — APPROVED** [review]

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
Ah, so the reason Taskcluster isn't running is probably because the branch is old and needs to be rebased. See https://github.com/web-platform-tests/wpt-pr-bot/issues/81 for tooling that might make this easier.

**Comment by foolip** [comment]
I've rebased the branch now, just need to wait for all checks to pass. @litherum feel free to merge yourself when the button is green :)

**Comment by foolip** [comment]
Results are in now: https://wpt.fyi/results/css/css-fonts?label=pr_head&max-count=1&pr=19917

@litherum the status seems to be the same for all browsers, is that what you expected?

wpt-chrome-dev-stability is [failing on Taskcluster](https://community-tc.services.mozilla.com/tasks/UcwM9UnGSQ2JGy6dCTeV8w) like this:

## Slow tests ##
|                     Test                     | Result | Longest duration (ms) | Timeout (ms) |
|----------------------------------------------|--------|-----------------------|--------------|
| `/css/css-fonts/standard-font-family-7.html` | `PASS` | `12290`               | `10000`      |

I don't know why that test would be slow, it doesn't do anything async even. @zcorpan FYI that the slow test check can give mysterious results.

I'll admin merge this (to override failing Taskcluster) once @litherum can confirm the results are as expected.

**Comment by litherum** [comment]
> @litherum the status seems to be the same for all browsers, is that what you expected?

Yes. My implementation hasn't made it into Safari Technology Preview yet.

**Comment by foolip** [comment]
Admin merging now, thanks for confirming @litherum!

**Pull request merged**